src/_data/how_to/go_contactless.yml

```diff
@@ -63,4 +63,5 @@ toc:
       - title: Maintenance and support
         url: "maintenance-and-support"
         paragraphs:
-          - text: Cal-ITP is here to help! You can <a href="/contact">contact us</a> for assistance or send an email to <a href="mailto:hello@calitp.org">hello@calitp.org</a>.
+          - text: Cal-ITP is here to help! Please download and review our <a href="https://docs.google.com/presentation/d/e/2PACX-1vQfJP8Qqnv3lpE8hysU4_rJNVzsuvINffvod6z6uQ9T_eKXaNNsDU4VHVeJkJtmvw/pub?start=false&loop=false&delayms=3000" target="_blank">Support Framework</a> - a document you can fill in and use to provide information to your internal team(s) about roles and responsibilities, common troubleshooting steps, how to manage your vendor relationships and how to monitor key performance outcomes.
```

Comment:
I linked to a published version of the Power Point on Google Drive: https://docs.google.com/presentation/d/e/2PACX-1vQfJP8Qqnv3lpE8hysU4_rJNVzsuvINffvod6z6uQ9T_eKXaNNsDU4VHVeJkJtmvw/pub?start=false&loop=false&delayms=3000#slide=id.p1 -- but that does not allow someone to download it.

But should it go to https://docs.google.com/presentation/d/1J_4TvF5fhI73fE4xcGPQ6e6atehFyEIx/edit#slide=id.p1 ?? That's a private restricted link. Can you change the link to whatever it should be and let me know?

<img width="1511" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/70f16481-0cd2-4d80-a2c4-253caa13a7f1">

Reply:
FWIW the PDF has parts like this that are intended for people to edit: 
<img width="1078" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/8fc64813-cf13-45e6-a239-72d9b830ad78">
<img width="1437" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/6510444a-703e-4b37-8c09-58777a9d2b53">
<img width="1324" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/88e925b5-612e-4d12-be36-15a09552624a">

If PDF, needs accessibility checks. Not sure if that color contrast is high enough.